src/librustdoc/html/render/print_item.rs

```diff
@@ -1419,6 +1422,9 @@ fn item_enum(w: &mut Buffer, cx: &mut Context<'_>, it: &clean::Item, e: &clean::
             if let clean::VariantKind::Tuple(ref s) = variant_data.kind {
                 write!(w, "({})", print_tuple_struct_fields(cx, s));
             }
+            if variant.is_default() {
+                w.write_str(" <span class=\"code-attribute\">(Default)</span>");
+            }
```

Comment:
In my opinion, this is too much, the attribute above suffices.

```suggestion
```

The property of being the *default value* of a type might be important but it is not so important that it warrants such a big tag. Further, it departs from Rustdoc's current style and claims to be a *code-attribute* which it isn't.

Reply:
I tried to copy the non-exhaustive behavior. For that, we show it in the code snippet and also the doc, with `(Non-exhaustive)` phrase.